priv/repo/migrations/20160615225508_default_trigger_timeout.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Cog.Repo.Migrations.DefaultTriggerTimeout do
+  use Ecto.Migration
+
+  def change do
+    alter table(:triggers) do
+      modify :timeout_sec, :integer, null: false, default: 30
```

Comment:
Unrelated: this is how long a trigger will hang around waiting for the pipeline to complete before finalizing the HTTP response. This is already the default in the code, but we didn't actually have a non-null constraint here in the database; I added it here while I was in the neighborhood.